src/utils/utils.ts

```diff
@@ -179,7 +179,7 @@ export function getProperty (value: { [key: string]: any}, key: string): any {
   }
   
   const keys: string[] = key.split('.');
-  let result: any = value[keys.shift()];
+  let result: any = value[keys.shift()!];
```

Comment:
I quickly compared my commit to yours and as expected your version still throws with

```/src/utils/utils.ts (182,21): An index expression argument must be of type 'string', 'number', 'symbol', or 'any'.```

The ! in this line indicates that `keys.shift()` can be treated as always defined (which is not absolutely accurate but as `value[undefined] =>undefined` which is handled below this might be sufficient).